config/environments/production.rb

```diff
@@ -41,7 +41,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  config.force_ssl = false
```

Comment:
We had some problems getting Docker to work locally over https, so we modified this for the time being while we were figuring out how Klaxon works locally. (See my other comment about this PR being very much in progress.)